ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3600,6 +3600,21 @@ AHU_DFSS_DXZC_ECONZ_EFSS2X_HT2ZC_CO2C:
   - EFSS2X
   - HT2ZC
   - CO2C
+  
+AHU_SFVSC_CHWRC_OAFC_SSPC_RAVOCC_RACO2C_ESFM:
+  description: "Single zone AHU with CO2, VOC based outside air control, static pressure based VFD speed control,return air temperature based Chilled water valve modulation."
+  is_canonical: true
+  implements:
+  - AHU
+  - SFVSC
```

Comment:
If this is truly a single-zone unit, you want to use `discharge` instead of `supply`

Reply:
Yes . We can use supply when the AHU is supplying it to VAVs right?